features/bootstrap/DatasetViewContext.php

```diff
@@ -416,7 +416,12 @@ public function iHaveSetPagesizeToOn($arg1, $arg2)
         }
     }
 
-
-
-
+    /**
+     * @Then There is a meta tag :arg1 with value :arg2
+     */
+    public function thereIsAMetaTagWithDOI($arg1, $arg2)
```

Comment:
The name of this function looks strange and should be changed to correspond with the `@then` feature step. What if the content of the `<meta>` tag doesn't contain a DOI? For example, your `@ok @issue-85 @issue-515` behat test contains this line:
```
And There is a meta tag "robots" with value "index"
```